_posts/2021-11-24-sprint-report-2.md

```diff
@@ -0,0 +1,62 @@
+---
+author: Tyler Weaver
+comments: false
+date: 2021-11-24 17:00:00+07:00
+layout: post
+slug: sprint-report-2
+title: MoveIt Sprint Report
+description: Nov 5 - Nov 25
+media_type: image
+media_link: /assets/images/blog_posts/2021_4_15_noetic_update.png
```

Comment:
This line was just copied from another blog post.  Please replace this when you add an image.